api/composer.json

```diff
@@ -6,6 +6,7 @@
     "license": "AGPL-3.0",
     "require": {
         "php": "^8.1",
+        "alphagov/notifications-php-client": "^5.0",
```

Comment:
I know this is part of GC Notify's official [documentation](https://documentation.notification.canada.ca/en/clients.html) for client libraries, but are any of these features ones we will require that are not available using the UK's PHP client?

> Feature differences to keep in mind
> - [Sending files by email is different with GC Notify](https://documentation.notification.canada.ca/en/send.html#sending-files-by-email-is-an-API-only-feature)
> - Sending bulk notifications through the GOV.UK Notify clients is not directly supported. You can still use one if the client supports customization of the URL endpoint
> - Sending letters is not available
> - Receiving text messages is not available

Reply:
Yeah, I'm willing to take this further and write a custom client so we can implement the missing features. I don't think it would be super complex and since this was so quick to implement we still have plenty of time in the 3 day timebox.

@tristan-orourke What are your thoughts on this?